### PR TITLE
Replace deprecated single-segment ns w Potemkin's import-vars

### PR DESCRIPTION
**Pull request by KingMob**
Obviously, this introduces a dependency on Potemkin, but Aleph already needs it, and Potemkin has barely changed in years.

### CODE REVIEW

**Comment by KingMob** [comment]
Fixes #65

**Comment by KingMob** [comment]
@p-himik can you try this out?

File: src/byte_streams.clj [hunk 1]
```diff
@@ -6,1019 +6,72 @@
...
-  (let [content (doto (ms/stream)
-                  (ms/put! (clojure.core/byte-array 5))
-                  (ms/close!))]
-    (convert content (seq-of java.nio.ByteBuffer)) ; this works
-    (= content (convert content (seq-of java.nio.ByteBuffer))))
 
+  ;; compare the namespaces, make sure we got everything
+  (require '[clojure.set :as set])
+
+  (set/difference
```
Comment (arnaudgeiser) [inline src/byte_streams.clj:67]: Maybe this can be turned into a Clojure test and the whole comment block can be removed.

**Review by arnaudgeiser — APPROVED** [review]
It's looking good. 
It means we'll have to release a new version of Aleph with this change.
It was also causing us some issues, but as we were aware of the work that has been done on `byte-streams`, it was easy to do the migration on our side.

**Comment by KingMob** [comment]
Fair point. I held off because I was debating whether to kill the top-level
test namespaces entirely. Insofar as they test behavior, there's now no way
the top-level tests can fail but the clj-commons tests will succeed. But
they *do* still test that the fns are available and callable.

Even if I kill the other tests, I suppose I should keep that one around

**Review by arnaudgeiser — APPROVED** [review]

**Pull request merged**